### PR TITLE
Adds few UI modification

### PR DESCRIPTION
**Pull request by pratap0007**
This patch adds:
- Adds Tooltip to resource catalog and kind icon on resource card
- Adds Tooltip to average rating on resource details page
- Reduce space between rating and rating icon on resource details page
- Replace CRA icon(browser icon) with Tekton logo
- Moves alert a little bit down
- Modifies resource description page width

Signed-off-by: Shiv Verma <shverma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/components/Cards/index.tsx [hunk 1]
```diff
@@ -35,18 +37,20 @@ const Cards: React.FC<Props> = (resources) => {
...
-
-                <span className="hub-catalog-icon">
-                  <Icon
-                    id={resource.catalog.icon}
-                    size={IconSize.sm}
-                    label={resource.catalog.name}
-                  />
-                </span>
-
+                <Tooltip content={<b> {resource.kind.name}</b>}>
```
Comment (PuneetPunamiya) [inline ui/src/components/Cards/index.tsx:40]: Can we please create a generic component of Tooltip and reuse it

File: ui/src/containers/App/App.css [hunk 2]
```diff
@@ -2,3 +2,7 @@
   position: fixed;
   width: 100%;
 }
+
+.pf-c-alert {
```
Comment (PuneetPunamiya) [inline ui/src/containers/App/App.css:6]: Can we please use className here and not use generic styles

File: ui/src/components/TooltipDisplay/TooltipDisplay.test.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    const component = shallow(<TooltipDisplay id={Icons.Build} name="Task" />);
+    expect(component.debug()).toMatchSnapshot();
+
+    const propsChildren = component.find(Tooltip).props().children;
+    assert(propsChildren);
+    expect(propsChildren.props['label']).toBe('Task');
+  });
+
+  it('should render Tooltip for resource catalog icon', () => {
+    const component = shallow(<TooltipDisplay id={Icons.User} name="Catalog" />);
```
Comment (vinamra28) [inline ui/src/components/TooltipDisplay/TooltipDisplay.test.tsx:19]:
```suggestion
    const component = shallow(<TooltipDisplay id={Icons.User} name="Tekton" />);
```

File: ui/src/components/TooltipDisplay/TooltipDisplay.test.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    expect(component.debug()).toMatchSnapshot();
+
+    const propsChildren = component.find(Tooltip).props().children;
+    assert(propsChildren);
+    expect(propsChildren.props['label']).toBe('Catalog');
```
Comment (vinamra28) [inline ui/src/components/TooltipDisplay/TooltipDisplay.test.tsx:24]:
```suggestion
    expect(propsChildren.props['label']).toBe('Tekton');
```

**Comment by PuneetPunamiya** [comment]
@pratap0007  Can we please add a tooltip on details page as well

File: ui/src/components/Description/Description.css [hunk 5]
```diff
@@ -1,5 +1,4 @@
 .hub-description {
-  max-width: 90em;
```
Comment (vinamra28) [inline ui/src/components/Description/Description.css:2]: do we need to remove this?

Comment (pratap0007) [reply ui/src/components/Description/Description.css:2]: I guess we need it

**Comment by PuneetPunamiya** [comment]
@pratap0007  Thank you for the patch, the pr looks fine to me :call_me_hand: 
/lgtm
/meow

**Pull request merged**